### DISCUSSION TITLE
FAQ: How do I install the mobile module?

### DISCUSSION THREAD

**Discussion by google-meet-meat** [pterodactyl/panel#2820] [General Community]
To install support for the Pterodactyl mobile app, run these commands in your panel directory:
```
composer config repositories.cloud composer https://packages.pterodactyl.cloud
composer require pterodactyl/mobile-addon --update-no-dev --optimize-autoloader
php artisan migrate
```
The Pterodactyl mobile app only has support for 0.7 panels. 1.0 has not added support.

**Comment by TekExplorer** [comment]
Note: This does not work for 1.0+ right now! this is only for 0.7 panels.

**Reply by google-meet-meat** [reply]
> Note: This does not work for 1.0+ right now! this is only for 0.7 panels.

Thanks, I've included that in my post.
